src/ui/features/xp-drop/XpDropClaim/XpDropClaim.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NavigationTitle } from 'src/ui/components/NavigationTitle';
+import { PageColumn } from 'src/ui/components/PageColumn';
+import { PageTop } from 'src/ui/components/PageTop';
+import { Frame } from 'src/ui/ui-kit/Frame';
+
+export function XpDropClaim() {
+  return (
```

Comment:
Is this a finished component?

Reply:
oh, I thought I deleted this file, but deleted the other one, thanks! deleted ✅